src/components/download/helperFunctions.js

```diff
@@ -107,6 +108,37 @@ const treeToNexus = (tree, colorings, colorBy, temporal) => {
   ].join("\n");
 };
 
+/**
+ * Create a properly formatted TSV string for given data using Papa.unparse()
+ * The optional columns parameter allows you to specify the specific keys to
+ * use as columns in the TSV string. Note, order of column names will
+ * determine order of output columns in the TSV string.
+ *
+ * If columns are not specified, then parser will use the keys of the first
+ * Object in the data array as the columns for the TSV string.
+ *
+ * See Papa Parse docs for more details about config options: https://www.papaparse.com/docs#json-to-csv
+ *
+ * @param {Array<Object>} data
```

Comment:
[not blocking] if a data element is `{key: <Array>}`, as it is for the "strains" column in `authorTSV`, then the array is stringified via `array.join(",")` (at least that what seems to be happening), but I couldn't see this in the linked docs. Would be nice to add this to the docstring here.

Reply:
Ah good point! I did a little digging in the papaparse source code and this happens because they call [`str.toString()` for all values](https://github.com/mholt/PapaParse/blame/824bbd9daf17168bddfc5485066771453cab423e/papaparse.js#L464) in the data object.

I'll add a note a here that all values will be converted to the string representation of the object.